docs/mlsys21/a100_throughputs.md

```diff
@@ -105,28 +105,35 @@ git clone https://github.com/UofT-EcoSystem/hfta.git
 cd hfta
 ```
 
-### Acquiring NVIDIA Nsight Compute CLI and DCGM
+### Acquiring NVIDIA Nsight Systems CLI and DCGM
 
-We require two installation files (`.deb`) for Nsight Compute and DCGM pre-downloaded to build the docker image.
+We require two installation files (`.deb`) for Nsight Systems and DCGM pre-downloaded to build the docker image.
 
-- Nsight Compute: version `3.1.72`, downloaded under `third_party/nsys/nsys_cli_2020.3.1.72.deb`
+- Nsight System: version `3.1.72`, downloaded under `third_party/nsys/nsys_cli_2020.3.1.72.deb`
 - DCGM version: version `2.0.10`, downloaded under  `third_party/dcgm/datacenter-gpu-manager_2.0.10_amd64.deb`
 
 In order to download the `.deb` files, you need to register a NVIDIA developer account via: <https://developer.nvidia.com/login>, after that, you can download the .deb file:
-- NVIDIA Nsight Computer at: <https://developer.nvidia.com/nsight-compute>
+- NVIDIA Nsight Systems CLI at: <https://developer.nvidia.com/gameworksdownload#?dn=nsight-systems-2020-3> (Select the "Linux CLI Only" option)
 - NVIDIA DCGM at: <https://developer.nvidia.com/dcgm>
 
 
 ### Download and launch docker image
 
 Follow the commands below to prepare and launch the docker image, this will take approximately 10 mins
 
-#### Build docker image
+#### Build or pull docker image
+
+You can either build the image from scratch or pull the pre-built image from docker-hub
+To build the image from scracth:
 ```bash
-# build the image, select from native1.6-cu10.2, nvidia20.06, nvidia20.08
+# build the image, select from nvidia20.06 or nvidia20.08 to ensure it works with A100
```

Comment:
Let's just limit it to `nvidia20.06` for the sake of simplicity, then type out the command explicitly below:
```bash
bash docker/build.sh nvidia20.06
```